optuna/visualization/_edf.py

```diff
@@ -102,30 +109,43 @@ def objective(trial, low, high):
 
     _imports.check()
 
-    if isinstance(study, Study):
-        studies = [study]
-    else:
-        studies = list(study)
+    layout = go.Layout(
+        title="Empirical Distribution Function Plot",
+        xaxis={"title": target_name},
+        yaxis={"title": "Cumulative Probability"},
```

Comment:
Possibly `_EDFInfo` contains these values too.

Reply:
Those are just literals, so imo it's fine to keep them as they are.